libkbfs/crypto_common.go

```diff
@@ -294,15 +294,15 @@ func (c CryptoCommon) DecryptPrivateMetadata(
 
 const minBlockSize = 256
 
-// nextPowerOfTwo returns next power of 2 greater than the input n.
+// thisPowerOfTwo returns next power of 2 greater than or equal to the input n.
 // https://en.wikipedia.org/wiki/Power_of_two#Algorithm_to_round_up_to_power_of_two
-func nextPowerOfTwo(n uint32) uint32 {
-	if n < minBlockSize {
+func thisPowerOfTwo(n int) int {
```

Comment:
Yes it's idempotent if it's already power of two; I thought that was there to save a couple instructions? New name sounds good!